test/osiris_log_SUITE.erl

```diff
@@ -1053,6 +1124,111 @@ offset_tracking_snapshot(Config) ->
     osiris_log:close(S3),
     ok.
 
+many_segment_overview(Config) ->
+    Data = crypto:strong_rand_bytes(1000),
+    EpochChunks =
+    [{1, [Data || _ <- lists:seq(1, 8)]} || _ <- lists:seq(1, 1024)] ++
+    [{2, [Data || _ <- lists:seq(1, 8)]} || _ <- lists:seq(1, 1024)] ++
+    [{3, [Data || _ <- lists:seq(1, 8)]} || _ <- lists:seq(1, 1024)] ++
+    [{4, [Data || _ <- lists:seq(1, 8)]} || _ <- lists:seq(1, 1024)] ++
+    [{5, [Data || _ <- lists:seq(1, 8)]} || _ <- lists:seq(1, 1024)],
+    Conf0 = ?config(osiris_conf, Config),
+    Conf = Conf0#{max_segment_size_bytes => 64000},
+    osiris_log:close(seed_log(Conf, EpochChunks, Config)),
+    %% {40051,{{0,40959},[{1,8184},{2,16376},{3,24568},{4,32760},{5,40952}]}}
+    {OverviewTaken, LogOverview} = timer:tc(fun () ->
+                                                    osiris_log:overview(maps:get(dir, Conf))
+                                            end),
+    ct:pal("OverviewTaken ~p", [OverviewTaken]),
```

Comment:
Can we not introduce new `ct:pal/2` log output, please?

Reply:
This test doesn't really have any assertions and the time taken is the only interesting part of this "test" so not sure what do do here. We can remove the test but it could be useful for future further optimisations.